modelci/ui.py

```diff
@@ -26,21 +27,22 @@ def single_model_view(model: Optional[dict], top=False):
         return ''
 
     if top:
-        model_name = f'[bold]{model["name"]}[/bold]'
+        model_name = f'[bold]{model["architecture"]}[/bold]'
     else:
-        model_name = model['name']
+        model_name = model['architecture']
 
+    # TODO reorganize and format and the parameters
     return (
         model['id'],
         model_name,
-        model['framework'],
-        model['engine'],
+        Framework(model['framework']).name,
```

Comment:
What if you pass in an MLModel? So you can do things like `model.id`, `model.framework.name`, etc.